lib/Installation/InstallationSettings/InstallationSettingsPage.pm

```diff
@@ -57,6 +57,18 @@ sub is_shown {
     return $self->{txt_overview}->exist();
 }
 
+sub is_loaded_completely {
+    my ($self) = @_;
+    my $result;
+    eval {
```

Comment:
so you silent the method with the `eval` I think this was a no-go, only should be used in the client itself, other contributors claimed it was bad practice in Perl, it doesn't behave so well as try/catch in other language (some modern languages even avoid try/catch as a build-in feature of the language).

I think it would be easier just to use the `is_shown` and add to `wait_until` a non-failure feature so we do the hack at the inner level, the only problem is the error if we die or not to use it for what I can see ... wdyt?